src/main/java/core/basesyntax/Animal.java

```diff
@@ -0,0 +1,65 @@
+package core.basesyntax;
+
+import java.util.Objects;
+
+public class Animal {
```

Comment:
let's make this class abstract, since we're not interested in creating object of abstract animal, we will just store common code for Animal subclasses